src/app/public/modules/title/title.service.spec.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  SkyAppTitleService
+} from './title.service';
+
+describe('Title service', () => {
+
+  let titleSvc: SkyAppTitleService;
+  let testTitle: string;
+
+  beforeEach(() => {
+    titleSvc = new SkyAppTitleService({
```

Comment:
There's no significant cost to instantiating it before each test, and the complexity of the code would be the same whether it's in a `beforeAll` or a `beforeEach`. And while this service doesn't currently have any state, it may in the future, in which case this code wouldn't need to be refactored.